src/Parsers/ASTLiteral.h

```diff
@@ -15,7 +15,7 @@ namespace DB
 class ASTLiteral : public ASTWithAlias
 {
 public:
-    explicit ASTLiteral(Field && value_) : value(value_) {}
+    explicit ASTLiteral(Field && value_) : value(std::move(value_)) {}
```

Comment:
Replace both ctrs to single with passing by value?

Reply:
Good point, updated.